### PR TITLE
PXB-2588 Update Restore Individual Tables (2.4)

### PR DESCRIPTION
**Pull request by patrickbirch**
Modified file:
source/xtrabackup_bin/restoring_individual_tables

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/restoring_individual_tables.rst [hunk 1]
```diff
@@ -86,14 +80,14 @@ Importing the Table
 
 On the destination server running |Percona Server| with |XtraDB| and
 `innodb_import_table_from_xtrabackup <http://www.percona.com/doc/percona-server/5.5/management/innodb_expand_import.html#innodb_import_table_from_xtrabackup>`_
-option enabled, or |MySQL| 5.6, create a table with the same structure, and
+option enabled, or |MySQL| 5.7, create a table with the same structure, and
 then perform the following steps:
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/xtrabackup_bin/restoring_individual_tables.rst]: We don't have innodb_import_table_from_xtrabackup
```suggestion
On the destination server, create a table with the same structure, and
then perform the following steps:
```

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/restoring_individual_tables.rst [hunk 2]
```diff
@@ -4,13 +4,12 @@
...
+In server versions prior to 5.7, it was not possible to copy tables between
 servers by copying the files, even with :term:`innodb_file_per_table`. However,
 with |Percona XtraBackup|, you can export individual tables from any |InnoDB|
-database, and import them into |Percona Server| with |XtraDB| or |MySQL| 5.6.
-(The source doesn't have to be |XtraDB| or |MySQL| 5.6, but the destination
-does.) This only works on individual :term:`.ibd` files, and cannot export a
-table that is not contained in its own :term:`.ibd` file.
+database, and import them into |Percona Server| with |XtraDB| or |MySQL| 5.7.
+(The source doesn't have to be |XtraDB| or |MySQL| 5.7, but the destination
+does.) This operation only works on individual :term:`.ibd` files. A table that is not contained in its own :term:`.ibd` file cannot be exported.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/xtrabackup_bin/restoring_individual_tables.rst]:
```suggestion
With |Percona XtraBackup|, you can export individual tables from any |InnoDB|
database, and import them into |Percona Server| with |XtraDB| or |MySQL| 5.7.
(The source doesn't have to be |XtraDB| or |MySQL| 5.7, but the destination
does.) This operation only works on individual :term:`.ibd` files. A table that is not contained in its own :term:`.ibd` file cannot be exported.
```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/restoring_individual_tables.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xtrabackup_bin/restoring_individual_tables.rst]: Updated.

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/restoring_individual_tables.rst (hunk 2 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xtrabackup_bin/restoring_individual_tables.rst]: Updated.

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**